patches/0003-dhcp-dhcp6-Ensure-that-DHCP-DHCPv6-DNS-server-update.patch

```diff
@@ -59,12 +59,12 @@ index 6da9cd6b..70f944b9 100644
  #endif /* LWIP_DHCP_PROVIDE_DNS_SERVERS */
  }
 diff --git a/src/core/ipv6/dhcp6.c b/src/core/ipv6/dhcp6.c
-index 002f688d..b18d6e9a 100644
+index dbc02be..8072e81 100644
 --- a/src/core/ipv6/dhcp6.c
 +++ b/src/core/ipv6/dhcp6.c
 @@ -538,16 +538,19 @@ dhcp6_handle_config_reply(struct netif *netif, struct pbuf *p_msg_in)
  
-     ip_addr_set_zero_ip6(&dns_addr);
+     memset(&dns_addr, 0, sizeof(dns_addr));
```

Comment:
hmm there is something strange with ordering, because this came from dhcpv6-dns-fix-v2.patch
I did the `git am` manually on the first two patches and simply `patch` the 3rd one, and this is what it came up with